com.orsan.Turtlico.json

```diff
@@ -0,0 +1,68 @@
+{
+    "app-id": "com.orsan.Turtlico",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "command": "turtlico",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--filesystem=host",
+        "--talk-name=ca.desrt.dconf",
+        "--talk-name=org.freedesktop.Flatpak",
```

Comment:
Why it needs to talk to Flatpak here?

Reply:
Because it will access GPIO from RPI etc